go.mod

```diff
@@ -2,4 +2,7 @@ module k8s.io/klog/v2
 
 go 1.13
 
-require github.com/go-logr/logr v1.2.0
+require (
+	github.com/go-logr/logr v1.2.0
+	k8s.io/utils v0.0.0-20220210201930-3a6ce19ff2f9
```

Comment:
this adds a circular dependency... utils already depends on klog, we should avoid making klog also depend on utils

Reply:
I think the test is useful enough that we should not remove it and instead carry a copy of the clock code -> https://github.com/kubernetes/klog/pull/310